### PR TITLE
Exposes Here provider to the bundle

### PR DESCRIPTION
**Pull request by DamienHarper**
Here provider was available in geocoder library but was not available from the Symfony bundle

### CODE REVIEW

**Comment by DamienHarper** [comment]
@Nyholm when do you think you'll have time to review this PR and release a new version of the bundle?
Right now the build fails but it is not related with the content of this PR

**Comment by DamienHarper** [comment]
Up

**Comment by norkunas** [comment]
Could you force push to rerun travis?

File: ProviderFactory/HereFactory.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        $resolver->setDefaults([
+            'httplug_client' => null,
+            'app_id' => null,
+            'app_code' => null,
+            'use_cit' => false,
+        ]);
+
+        $resolver->setAllowedTypes('httplug_client', ['object', 'null']);
+        $resolver->setAllowedTypes('app_id', ['string', 'null']);
```
Comment (norkunas) [inline ProviderFactory/HereFactory.php]: Looking at the https://github.com/geocoder-php/here-provider/blob/master/Here.php `app_id` couldn't be null.

File: ProviderFactory/HereFactory.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        $resolver->setAllowedTypes('app_code', ['string', 'null']);
```
Comment (norkunas) [inline ProviderFactory/HereFactory.php]: Same here

**Comment by norkunas** [comment]
But the `app_id` and `app_code` should be marked as required, no? :slightly_smiling_face:

**Comment by norkunas** [comment]
Also could you add add this provider to composer `require-dev` and add tests to include it in `Bazinga\GeocoderBundle\Tests\Functional\ProviderFactoryTest` ?

**Comment by DamienHarper** [comment]
@norkunas it should be good right now, let me know

**Comment by norkunas** [comment]
Thank you @DamienHarper

**Pull request merged**